bin/hubot-slack-github-issues

```diff
@@ -0,0 +1,74 @@
+#! /usr/bin/env node
+
+var packageInfo = require('../package.json');
+var Config = require('../lib/config');
+var fs = require('fs');
+var path = require('path');
+var programName = path.basename(process.argv[1]);
+var option = process.argv[2];
+var usage = [
+  'Usage:',
+  '  ' + programName + ' [-hv|print-template]',
+  '  ' + programName + ' validate <config.json>',
+  '',
+  'Where:',
+  '  -h, --help, help        show this help',
+  '  -v, --version, version  show version info',
+  '  print-template          print an example slack-github-issues.json config',
+  '  validate <config.json>  validate the specified config file',
+].join('\n');
+
+var showHelp, showVersion;
+
+if (option === 'print-template') {
+  var config = require('../config/slack-github-issues.json');
+  console.log(JSON.stringify(config, null, 2));
+  process.exit(0);
+}
+
+showHelp = (option === '-h' || option === '--help' || option === 'help');
+showVersion = (option === '-v' || option === '--version' ||
+  option === 'version');
+
+if (showHelp || showVersion) {
```

Comment:
I don't really think this needs to be a nested if, rather than two separate ifs, for the two options, but I'm ok with it.